oraclizeAPI_0.4.sol

```diff
@@ -342,6 +342,11 @@ contract usingOraclize {
             oraclize_setNetworkName("eth_rinkeby");
             return true;
         }
+        if (getCodeSize(0xa2998EFD205FB9D4B4963aFb70778D6354ad3A41)>0){ //goerli testnet

```

Comment:
Like the consistency with the extant formatting, even though I hate the extant formatting!